core/src/main/java/org/apache/accumulo/core/client/BatchWriterConfig.java

```diff
@@ -309,6 +311,36 @@ public boolean equals(Object o) {
     return false;
   }
 
+  public BatchWriterConfig merge(BatchWriterConfig other) {
+    BatchWriterConfig result = new BatchWriterConfig();
+    if (this.maxMemory != null) {
```

Comment:
Could do something like the following here

```java
private static<T> T merge(T o1, T o2) {
  if(o1 != null) return o1;
  return o2;
}

public static BatchWriterConfig merge(BatchWriterConfig other) {
   BatchWriterConfig result = new BatchWriterConfig();
   result.maxMemory = merge(this.maxMemory, other.maxMemory);
}
```

Reply:
This was fixed